migrations/20160519162112-make-group.slug-unique-migration.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = {
+  up: function (queryInterface) {
+    return queryInterface.addIndex('Groups', ['slug'], {
```

Comment:
yes, unfortunately. Wasn't able to find any other way of doing this. If you do combination of iD and slug, that means that slug can repeat (because ID is always unique). And we tie slug directly to serve the group content.